app/src/main/java/com/android/gatherly/utils/GatherlyAlertDialog.kt

```diff
@@ -33,52 +44,84 @@ fun GatherlyAlertDialog(
       },
       text = {
-        Text(
-            text = bodyText,
-            textAlign = TextAlign.Center,
-            modifier = Modifier.testTag(AlertDialogTestTags.BODY))
-      },
-      dismissButton = {
-        Button(
-            colors =
-                buttonColors(
-                    containerColor = MaterialTheme.colorScheme.primaryContainer,
-                    contentColor = MaterialTheme.colorScheme.onPrimaryContainer),
-            onClick = onDismiss,
-            modifier = Modifier.testTag(AlertDialogTestTags.DISMISS_BTN)) {
-              Text(text = dismissText, color = MaterialTheme.colorScheme.onPrimaryContainer)
-            }
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+          Text(
+              text = bodyText,
+              textAlign = TextAlign.Center,
+              modifier = Modifier.testTag(AlertDialogTestTags.BODY))
+
+          neutralText?.let { text ->
+            Spacer(modifier = Modifier.height(16.dp))
+            Button(
+                colors =
+                    buttonColors(
+                        containerColor = MaterialTheme.colorScheme.primaryContainer,
+                        contentColor = MaterialTheme.colorScheme.onPrimaryContainer),
+                onClick = { onNeutral?.invoke() },
+                enabled = neutralEnabled,
+                modifier = Modifier.fillMaxWidth().testTag(AlertDialogTestTags.NEUTRAL_BTN)) {
+                  Text(text = text)
+                }
+          }
+        }
       },
-      onDismissRequest = onDismiss,
+      dismissButton = null,
+
+      // Entire button logic here, to get spacing
       confirmButton = {
-        Button(
-            colors =
-                buttonColors(
-                    containerColor =
-                        if (isImportantWarning) {
-                          MaterialTheme.colorScheme.surfaceVariant
-                        } else {
-                          MaterialTheme.colorScheme.primaryContainer
-                        },
-                    contentColor =
-                        if (isImportantWarning) {
-                          MaterialTheme.colorScheme.error
-                        } else {
-                          MaterialTheme.colorScheme.onPrimaryContainer
-                        }),
-            enabled = confirmEnabled,
-            onClick = onConfirm,
-            modifier = Modifier.testTag(AlertDialogTestTags.CONFIRM_BTN)) {
-              Text(
-                  text = confirmText,
-                  color =
-                      if (isImportantWarning) {
-                        MaterialTheme.colorScheme.error
-                      } else {
-                        MaterialTheme.colorScheme.onPrimaryContainer
-                      })
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+              // Dismiss Button
+              Button(
```

Comment:
Important: Is there a specific reason why the dismiss button is implemented inside the `confirmButton` logic?
If not, I strongly recommend moving the dismiss logic to the dedicated `dismissButton` parameter provided by `AlertDialog`.

```suggestion
              AlertDialog(
              [...]
    onDismissRequest = onDismiss,

    dismissButton = {
        Button([...]) { Text([...])}
     },

    confirmButton = {
        Button([...]) {
            Text([...])
        }
    }
)

```

This approach keeps the responsibilities clear:
`dismissButton` for cancel/close actions
`confirmButton` for validation
`onDismissRequest` for system-driven dismiss behavior

Reply:
Yes, it's a minor nitpick about how the alert dialog should look. We will fix this in the next sprint.